bazel/cargo/wasmtime/remote/BUILD.rustix-0.38.14.bazel

```diff
@@ -63,12 +63,12 @@ cargo_build_script(
         "cargo-raze",
         "manual",
     ],
-    version = "0.37.19",
+    version = "0.38.14",
     visibility = ["//visibility:private"],
     deps = [
-        "@wasmtime__cc__1_0_79//:cc",
-    ] + selects.with_or({
-        # cfg(all(any(target_os = "android", target_os = "linux"), any(rustix_use_libc, miri, not(all(target_os = "linux", any(target_arch = "x86", all(target_arch = "x86_64", target_pointer_width = "64"), all(target_endian = "little", any(target_arch = "arm", all(target_arch = "aarch64", target_pointer_width = "64"), target_arch = "powerpc64", target_arch = "riscv64", target_arch = "mips", target_arch = "mips64"))))))))
+        "@wasmtime__cc__1_0_83//:cc",
+    ]  + selects.with_or({
```

Comment:
I think the double-space here might be an issue. If not, then run `buildifier` and let it fix it for you.